Dart/dp/StringPermutations.dart

```diff
@@ -0,0 +1,74 @@
+import 'dart:io';
```

Comment:
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top